src/SprykerEco/Yves/FactFinderWebComponents/FactFinderWebComponentsConfig.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\FactFinderWebComponents;
+
+use Spryker\Yves\Kernel\AbstractBundleConfig;
+use SprykerEco\Shared\FactFinderWebComponents\FactFinderWebComponentsConstants;
+
+class FactFinderWebComponentsConfig extends AbstractBundleConfig implements WebComponentsConfigInterface
+{
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getCommunicationConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::COMMUNICATION_WIDGET_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getSearchBoxConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::SEARCH_BOX_WIDGET_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getBreadcrumbConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::BREADCRUMB_WIDGET_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getSearchButtonConfig(): array
+    {
+        return $this->get(FactFinderWebComponentsConstants::SEARCH_BUTTON_WIDGET_CONFIG);
+    }
+
+    /**
+     * @inheritdoc
+     *
+     * @return array
+     */
+    public function getRecordListConfig(): array
```

Comment:
Would be nice to name this name same as constant.

Reply:
Because we have Component instead of Widget I changed constants names from *_WIDGET_CONFIG to _COMPONENT_CONFIG, and changed getConfg() methods names either.